packages/now-node/src/index.ts

```diff
@@ -409,16 +409,12 @@ function isPortInfo(v: any): v is PortInfo {
   return v && typeof v.port === 'number';
 }
 
+const tscPath = resolve(require.resolve(eval('"typescript"')), '../bin/tsc');
```

Comment:
I'm curious, why eval() here if we are no longer using ncc?

Reply:
We are still using ncc on the main `src/index.ts`.

We're no longer using ncc to _bundle `typescript` itself, and instead preferring it as a regular dependency.

Reply:
If that's the case, then you'll need to exclude it from the ncc build with `-e typescript` like we do with `@now/build-utils` or else we'll have two copies of typescript.

Reply:
I don't think it's required (since the point of the `eval()` it to bypass the ncc bundling logic), but I've added in https://github.com/zeit/now/pull/4185/commits/561086e86ef7829d7046728ed666c21d62ad10ab anyways.